src/components/Heading/Heading.tsx

```diff
@@ -15,6 +16,7 @@ export interface HeadingProps extends StyledHeadingProps {
 }
 
 const StyledHeading = styled.header<StyledHeadingProps>`
+  color: ${props => (props.color ? props.color : Color.white)};
```

Comment:
I believe this should work:

```suggestion
  color: ${props => (props.color ?? Color.white)};
```